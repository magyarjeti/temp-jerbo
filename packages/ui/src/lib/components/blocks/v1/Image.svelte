<script lang="ts">
	import type { ImageBlock } from '@packages/api/sdk/v1/schema';
	import MediaItem from '../../MediaItem.svelte';

	interface Props {
		data?: ImageBlock;
	}

	let { data }: Props = $props();
</script>

<div>
	<MediaItem
		mediaItem={data?.mediaItem}
		alt={data?.params?.alt}
		caption={data?.content}
		showAuthor={data?.params?.showauthor}
		enableGalleryView
	></MediaItem>
</div>
