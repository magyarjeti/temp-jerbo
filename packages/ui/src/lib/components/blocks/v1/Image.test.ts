import { render } from '@testing-library/svelte';
import { expect, test } from 'vitest';
import Image from './Image.svelte';

test('Image snapshot', () => {
	const { container } = render(Image);
	expect(container).toMatchSnapshot();
});
