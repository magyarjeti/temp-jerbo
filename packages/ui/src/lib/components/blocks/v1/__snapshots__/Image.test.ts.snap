// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Image snapshot 1`] = `
<div>
  <figure>
     
  </figure>
</div>
`;
